pandas/_config/config.py

```diff
@@ -820,6 +821,31 @@ def inner(x):
     return inner
 
 
+def is_pos_int(value):
+    """
+    Verify that value is None or a positive int.
+
+    Parameters
+    ----------
+    value : None or int
+            The `value` to be checked.
+
+    Raises
+    ------
+    ValueError if the value is not equal to a positive integer, 0 or None.
```

Comment:
I think this should be formatted like

```
Raises
-------
ValueError
    When the value is not a positive integer or None
```

Reply:
Yes, 0 is allowed, and is necessary for existing internal code.  I think you are right, at this point "is_pos_int" is technically not an accurate name.  I'm fine with "is_nonnegative_int".  The docstring change seems fine to me as well.  I'll make the changes and do another push.